pact/Pact/Core/Repl/Compile.hs

```diff
@@ -17,6 +17,8 @@ module Pact.Core.Repl.Compile
  , interpretEvalBigStep
  , interpretEvalSmallStep
  , interpretEvalDirect
+ , interpretReplProgram'
```

Comment:
nit: Slight preference against `'` names. Renaming `interpretReplProgram'` to `interpretReplProgramWithBuiltinEnv`, or renaming `interpretReplProgram` to `interpretReplProgramDefaultBuiltins` is slightly clearer.

Reply:
I mean the thing is that it takes an `Interpreter` now, so probably should just be `interpretReplProgram`. The `'` version should be purely internal to the module.

Reply:
Fixed 👍